pages/app/Splash.js

```diff
@@ -17,14 +17,9 @@ import Header from 'components/splash/layout/Header';
 import { MARKERS } from 'utils/splash/Markers';
 
 // Components
+import GlobeCesium from 'pages/app/pulse/globe-cesium';
```

Comment:
Are you getting a component from another page?? As you need to share it between pages you should move this component to the components folder.